data/yaml/venues/c3nlp.yaml

```diff
@@ -0,0 +1,2 @@
+acronym: C3NLP
+name: Cross-Cultural Considerations in NLP @ EACL
```

Comment:
The venue entry shouldn't mention a particular parent conference (remove `@ EACL`)